### DISCUSSION TITLE
How to apply letsencrypt cert with certbot on Ubuntu 20.04?

### DISCUSSION THREAD

**Discussion by arvindm-tg** [requarks/wiki#7139] [Help / Questions]
Hello Everyone, 

I have installed wiki.js by following the instructions  from the below link on a Azure VM (Ubuntu 20.04)

https://docsbythenerd.com/en/installation-guides/installing-wikijs

I am facing difficulties in configuring the letsencrypt  cert for my site. Are there specific instructions for this method of installation for applying the cert?

Any help will be greatly appreciated. 

Thanks in advance

**Comment by NGPixel** [comment]
https://docs.requarks.io/install/ubuntu
